src/Aspire.Hosting.Azure.CosmosDB/AzureCosmosDBResource.cs

```diff
@@ -43,7 +43,10 @@ public class AzureCosmosDBResource(string name, Action<AzureResourceInfrastructu
     /// </summary>
     internal IAzureKeyVaultSecretReference? ConnectionStringSecretOutput { get; set; }
 
-    private BicepOutputReference NameOutputReference => new("name", this);
+    /// <summary>
+    /// Gets the "name" output reference from the bicep template for the Azure Cosmos DB resource.
+    /// </summary>
+    public BicepOutputReference NameOutputReference => new("name", this);
```

Comment:
Do we want to suffix these with `OutputReference`? We don't have a suffix for others. For example, above we have `ConnectionString` and `ConnectionStringOutput`.

I know just `Name` will conflict with the base Resource.Name property. But we could have `NameOutput`, or possibly a prefix like `AzureName`. I'd probably go with `NameOutput` I guess.